src/api/config/clock.rb

```diff
@@ -27,6 +27,10 @@ module Clockwork
     SendEventEmailsJob.perform_later
   end
 
+  every(5.minutes, 'measurements') do
+    MeasurementsJob.perform_later
```

Comment:
Now you definitely need to add a timestamp to your measurement because there is no guarantee that the queue is empty.

Reply:
Adding a timestamp to the measurement would not help to have measurements in time, to prevent possible delays caused by triggering jobs attached to not empty queues. However, I will try using a clockwork thread, instead of a delayed job.

Reply:
Ah no, it'll just be whenever the queue runs. okay